src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -80,6 +80,7 @@ class AnalyticsConstants {
             stringParam('CONFIG_BRANCH', '$ANALYTICS_CONFIGURATION_RELEASE', 'e.g. tagname or origin/branchname, or $ANALYTICS_CONFIGURATION_RELEASE')
             stringParam('CONFIG_REPO', 'git@github.com:edx/edx-analytics-configuration.git', '')
             stringParam('NOTIFY', allVars.get('NOTIFY','$PAGER_NOTIFY'), 'Space separated list of emails to send notifications to.')
+            stringParam('PYTHON_VERSION', env.get('PYTHON_VERSION', allVars.get('PYTHON_VERSION')), 'Python version that will be used to run pipelines task')
```

Comment:
Maybe better if the text described it as a python path, and that it's the version to run inside EMR:

`Path to a specific python interpreter inside the EMR cluster that will be used to run pipelines tasks.`

Reply:
Sure.